atlasdb-client/src/integrationInput/java/com/palantir/atlasdb/table/description/generated/SchemaApiTestV2Table.java

```diff
@@ -166,7 +167,7 @@ public Namespace getNamespace() {
 
     /**
      * Returns the value for column Column2 and specified row components. */
-    public Optional<String> getColumn2(String component1) {
+    public Optional<StringValue> getColumn2(String component1) {
```

Comment:
This does not compile for me:

```
/Users/hsaraogi/code/atlasdb/atlasdb-client/src/integrationInput/java/com/palantir/atlasdb/table/description/generated/SchemaApiTestV2Table.java:330: error: cannot find symbol
            StringValue newValue = processor.apply(result.get());
            ^
  symbol:   class StringValue
  location: class SchemaApiTestV2Table
33 errors
:atlasdb-client:compileIntegrationInputJava FAILED
```

Reply:
yep, not compiling in circle either. let me fix it